mibidata/mibi_image.py

```diff
@@ -459,4 +460,7 @@ def export_pngs(self, path, size=None):
             raise TypeError('Unsupported dtype: %s' % data.dtype.type)
         for i, label in enumerate(self.channels):
             im = converter(data[:, :, i])
-            skio.imsave(f'{os.path.join(path, label)}.png', im)
+            png_name = label[1] if isinstance(label, tuple) else label
```

Comment:
Can a label be a tuple of length 1? Or does the setter enforce that it's either a string or length 2?

Reply:
Looks like `_set_channels` checks to ensure each channel is a tuple of length 2.

https://github.com/ionpath/mibitracker-client/blob/71fa9cab7866038510021356b4bd3e9bbde9e4c0/mibidata/mibi_image.py#L185

Reply:
This is a bit of a contrived example, but with that you could set the channels to be 1-tuples:
```
im.channels = [('a',), ('b',), ('c',)]
```
This is an issue with the current channel-setter, your changes *should* have been fine, but maybe we want to enforce that a channel is a 2-tuple or a string here so that we can't have this incompatibility.